Makefile

```diff
@@ -50,3 +50,7 @@ test_JHU:
 	python -B tests/test_JHU.py 
 test_Merge:
 	python -B tests/test_Merge.py
+
+# run a python development server to work on the docs
+serve:
+	python -m http.server -d docs
```

Comment:
This is just for convenience, particularly so that you don't need to change directories to start the docs server